### PR TITLE
Added message in the case of Not Supported Exception in PublicExceptionMessage.cs

### PR DESCRIPTION
**Pull request by supragyan**
Added message in the case of Not Supported Exception in PublicExceptionMessage.cs
Added key and value in resource file.
Replaced NotImplementedException to NotSupportedException in DamSqlDBWithMultiShard class

### CODE REVIEW

File: root_VS2012/programs/C#/Frameworks/Infrastructure/Public/Db/DamSqlDbWithMultiShard/DamSqlDbWithMultiShard/DamSqlDbWithMultiShard.cs [hunk 1]
```diff
@@ -613,26 +613,22 @@ private void PreExecQuery()
 
         #region 外部公開API
 
-        /// <summary>Selectクエリを実行し、データテーブルを返す。</summary>
-        /// <param name="dt">データテーブル</param>
+        /// <summary>This method is not supported.</summary>
         /// <remarks>
```
Comment (daisukenishino) [inline root_VS2012/programs/C#/Frameworks/Infrastructure/Public/Db/DamSqlDbWithMultiShard/DamSqlDbWithMultiShard/DamSqlDbWithMultiShard.cs]: Remove the remarks tag itself.

File: root_VS2012/programs/C#/Frameworks/Infrastructure/Public/Resources/PublicExceptionMessageResource.ja-JP.resx [hunk 2]
```diff
@@ -180,6 +180,9 @@
   <data name="NOT_IMPLEMENTED" xml:space="preserve">
     <value>この処理は実装されていません。</value>
   </data>
+  <data name="NOT_SUPPORTED" xml:space="preserve">
+    <value>このプロセスは、サポートされていません。</value>
```
Comment (daisukenishino) [inline root_VS2012/programs/C#/Frameworks/Infrastructure/Public/Resources/PublicExceptionMessageResource.ja-JP.resx]:
```
このプロセスは、サポートされていません。
```
▼ ▼ ▼
```
この処理は、サポートされていません。
```

**Comment by daisukenishino** [comment]
@supragyan 

I've pointed out in this pull-request. Please modify this.

File: root_VS2012/programs/C#/Frameworks/Infrastructure/Public/Db/DamSqlDbWithMultiShard/DamSqlDbWithMultiShard/DamSqlDbWithMultiShard.cs (hunk 1 above)
Comment (supragyan) [inline root_VS2012/programs/C#/Frameworks/Infrastructure/Public/Db/DamSqlDbWithMultiShard/DamSqlDbWithMultiShard/DamSqlDbWithMultiShard.cs]: @daisukenishino 
We have removed remarks tag from ExecSelectFill_DT,ExecSelectFill_DS,ExecSelectScalar and ExecInsUpDel_NonQuery methods.

Please review.

File: root_VS2012/programs/C#/Frameworks/Infrastructure/Public/Resources/PublicExceptionMessageResource.ja-JP.resx (hunk 2 above)
Comment (supragyan) [inline root_VS2012/programs/C#/Frameworks/Infrastructure/Public/Resources/PublicExceptionMessageResource.ja-JP.resx]: @daisukenishino 

We have modified resource file key value in https://github.com/SymphonyTeleca/OpenTouryoTemplates/commit/8d6bed3bf10136a79713c52cd16fdd940a0a0d06#diff-77bff2350f46571a24459ecb0d556509R184

Please review.

**Comment by daisukenishino** [comment]
@supragyan 

Thank you for the quick response.

**Pull request merged**